README.md

```
@@ -50,7 +50,7 @@ The following props are accepted by the picker:
 | autoFocusSearch | `boolean` | `true` | Controls the auto focus of the search input. |
 | width | `number / string` | `350` | Controls the width of the picker. You can provide a number that will be treated as pixel size, or any accepted css width as string.
 | height | `number / string` | `450` | Controls the height of the picker. You can provide a number that will be treated as pixel size, or any accepted css width as string.
-| categoryHeight | `number / string` | `100` | Controls the height of the home page reaction category. You can provide a number that will be treated as pixel size, or any accepted css width as string.
+| categoryHeight | `number / string` | `95` | Controls the height of the home page reaction category. You can provide a number that will be treated as pixel size, or any accepted css width as string.
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)